src/Cli/ExtraCommands.php

```diff
@@ -49,6 +50,138 @@ public function repoConvertData($path, $options = ['format' => 'json'])
+            $number_of_lines_to_search = 5;
+            $first_empty_line = -1;
+            $last_badge_line = -1;
+            $badge_insert_line = -1;
+            foreach ($lines as $line_number => $line) {
+                if ($first_empty_line == -1 && empty(trim($line))) {
+                    $first_empty_line = $line_number;
+                }
+                // Is this line a badge?
+                if (preg_match('/\[\!\[[A-Za-z0-9\s]+\]\(.*\)/', $line)) {
+                    $last_badge_line = $line_number;
+                    // Is this line the License badge?
+                    if (preg_match('/\[\!\[License]\(.*\)/', $line)) {
+                        if ($line_number) {
+                            $badge_insert_line = $line_number - 1;
+                        } else {
+                            $badge_insert_line = 0;
+                        }
+                    }
+                } else {
+                    if ($last_badge_line != -1) {
+                        // We already found the badges, exit foreach.
+                        break;
+                    } elseif ($line_number > $number_of_lines_to_search) {
+                        // We've searched enough lines, exit foreach.
+                        break;
+                    }
+                }
+            }
+            if ($badge_insert_line === -1) {
+                if ($last_badge_line !== -1) {
+                    // If we found badges, we'll insert this badge after the last badge.
+                    $badge_insert_line = $last_badge_line + 1;
+                } elseif ($first_empty_line !== -1) {
+                    // If we didn't find any badges, we'll insert this badge at the first empty line.
+                    $badge_insert_line = $first_empty_line + 1;
+                } else {
+                    // Final fallback: insert badge in the second line of the file.
+                    $badge_insert_line = 1;
+                }
+            }
+            // Insert badge contents and empty line after it.
+            array_splice($lines, $badge_insert_line, 0, [$badge_contents, '']);
+            $readme_contents = implode("\n", $lines);
+            file_put_contents("$dir/README.md", $readme_contents);
+            $workingCopy->add("$dir/README.md");
+        }
+
+        $commit_message = $options['commit-message'];
+        $workingCopy->commit($commit_message);
+        $workingCopy->push('origin', $branchName);
+        $message = $options['pr-title'];
+        $body = $options['pr-body'];
+        $workingCopy->pr($message, $body, $baseBranch, $branchName);
+    }
+
+    /**
+     * Get right badge markdown.
+     */
+    protected function getSupportLevelBadge($level)
+    {
+        $badges = [
+            'ea' => '[![Early Access](https://img.shields.io/badge/pantheon-EARLY_ACCESS-yellow?logo=pantheon&color=FFDC28&style=for-the-badge)](https://github.com/topics/early-access?q=org%3Apantheon-systems)',
```

Comment:
If we put "Pantheon" badges / other Pantheon-specific info in, then this command will need to go in the update tool. If we make the user pass in info like the label & etc (or maybe even the whole markdown for the badge), then the command could stay here.